### PR TITLE
chore: change all instances of fast-dna to fast

### PR DESCRIPTION
**Pull request by EisenbergEffect**
## Description

This PR replaces instances of "FAST-DNA" or "fast-dna" with "FAST" and "fast" respectively. This is in preparation for the repo rename from "fast-dna" to "fast" as well as the rebrand.

There were several special cases that should be pointed out as part of this renaming:

* All references to `@microsoft/eslint-config-fast-dna` were left in tact since `fast-dna` is part of the published package name.
* I wasn't sure what to do with `res.cloudinary.com/fast-dna`. I've changed it to `res.cloudinary.com/fast` in this PR but could easily change it back. /cc @awentzel
* I changed code climate references from "fast-dna" to "fast" as well. Is that correct? /cc @awentzel 
* I changed our medium blog from `https://medium.com/fast-dna` to `https://medium.com/fast` Are we going to change that name or should I revert that? /cc @Falkicon @chrisdholt 

> **WARNING:** Do not merge this PR until after the repo as been renamed and the build process as been refreshed and passed.

### CODE REVIEW

File: sites/fast-creator/app/index.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ root.setAttribute("id", "root");
 document.body.appendChild(root);
 const style: HTMLStyleElement = document.createElement("style");
 style.innerText =
-    "@font-face { font-family: SegoeUIVF; src: url(https://res.cloudinary.com/fast-dna/raw/upload/v1558051831/SegoeUI-Roman-VF_web.ttf) format(\"truetype\"); font-weight: \"1 1000\"; } @font-face { font-family: Segoe UI; src: url('//c.s-microsoft.com/static/fonts/segoe-ui/west-european/Normal/latest.woff2') format('woff2'); } body, html { font-family: SegoeUIVF, Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif; font-size: 12px; padding: 0; margin: 0; }";
```
Comment (janechu) [inline sites/fast-creator/app/index.tsx:15]: presumably someone needs to update the files pointing to cloudinary

Comment (awentzel) [reply sites/fast-creator/app/index.tsx:15]: If this was the old CDN we used under Jason's account, then yes, this should be changed to use our own CDN, and place these files in source code `/site-utilities/statics/assets/fonts/*`

**Comment by awentzel** [comment]
@EisenbergEffect - Code Climate looks good, but it will break (stop reporting).  As the next phase, I'll have to add the "new" project to code climate and revalidate the badge guid's are still valid.

**Comment by EisenbergEffect** [comment]
Ok, I've reverted the cloudinary and medium changes. The code climate change is kept.

**Pull request merged**